dapps/digital-twin-detail/src/i18n/de.json

```
@@ -0,0 +1,34 @@
+{
+  "_twin-detail": {    
+    "data": {
+      "data-title": "Daten",
+      "general": {
+        "desc": "Beschreibung",
+        "general-title": "Allgemein",
+        "information-title": "Informationen",
+        "name": "Name",
+        "owner": "Besitzer",
+        "twin-image": "Zwilling Bild",
+        "type": "Typ"
+      },
+      "logs": {
+        "logs-title": "Logs"
+      },
+      "specifications": {
+        "specifications-title": "Spezifikationen"
+      }
+    },
+    "did": {
+      "did-title": "Decentralized ID"
```

I must admit it looks ugly in german ... shall we always use here the english word?

```suggestion
      "did-title": "Dezentralisierte ID"
```

---

This will likely change in the future anyways when we have a proper concept for wording to the user.